backend/sozisel/lib/sozisel_web/schema/resolvers/session_recording_resolvers.ex

```diff
@@ -0,0 +1,205 @@
+defmodule SoziselWeb.Schema.Resolvers.SessionRecordingResolvers do
+  alias SoziselWeb.Context
+  alias Sozisel.Repo
+  alias SoziselWeb.Schema.Helpers
+  alias SoziselWeb.Schema.Subscriptions.Topics
+  alias Sozisel.Model.Sessions
+  alias Sozisel.Model.Sessions.Session
+  alias Sozisel.Model.SessionRecordings
+  alias Sozisel.Model.SessionRecordings.SessionRecording
+  alias Sozisel.Model.Polls.PollResult
+
+  import SoziselWeb.Schema.Middleware.ResourceAuthorization, only: [fetch_resource!: 2]
+
+  require Logger
+
+  @media_storage_module Sozisel.MediaStorage.Disk
+
+  def get_session(_parent, %{id: session_id}, ctx) do
```

Comment:
Just like below

Reply:
I think that you keep those functions in `session_resolvers.ex` as well and forgot to delete it from here?

Reply:
Yeah, exactly